ssl.crt

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What is this file? Just a test example? If so, give a clearer name. Also, move it to the websocket folder.